src/components/Hscroll/index.jsx

```diff
@@ -33,9 +14,7 @@ const Hscroll = ({
   children,
   ...rest
 }) => {
-  const [isScrolled, setIsScrolled] = useState(false);
-  const [isAtScrollEnd, setIsAtScrollEnd] = useState(false);
-  const contentEl = useRef(null);
+  const [onScroll, contentEl, isScrolled, isAtScrollEnd] = useGetScrollAttributes();
```

Comment:
did we want to change `isScrolled` to `isAtScrollStart`? The value would be flipped, but it might be more clear